pyaedt/maxwell.py

```diff
@@ -1172,6 +1185,29 @@ def get_model_depth(self):
         else:
             return None
 
+    @model_depth.setter
+    def model_depth(self, value):
+        """Set model depth.
+
+        Example
+        -------
+
+        Set and get the model depth for a 2D Maxwell analysis.
+
+        >>> from pyaedt import Maxwell2d
+        >>> maxwell_2d = Maxwell2d()
+        >>> maxwell_2d.model_depth="90 mm"
+        >>> maxwell_2d.model_depth
+        0.09
+        """
+
+        try:
+            depth_value = float_units(value)
+        except:
+            depth_value = self.variable_manager[value].value
+
+        self.design_properties["ModelDepth"] = value
```

Comment:
@MaxJPRey this way is not gonna work. design_properties contains the .aedt file parsing information. It's a readonly information. You cannot update it from it. You need to check in AEDT what is the relative command to assign model depth.

Reply:
OK. The get was previously just extracting info from the aedt file. It seems that this property should be on the design. I will modify that.
Thanks @maxcapodi78